src/components/Navbar/NavLanguageSelector.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { CSSProperties } from 'react'
+import { Select } from 'antd'
+import { GlobalOutlined } from '@ant-design/icons'
+
+import { Languages } from 'constants/languages/language-options'
+
+// Language select tool seen in top nav
+export default function LanguageSelector() {
+  const selectStyle: CSSProperties = {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'space-evenly',
+    cursor: 'pointer',
+    height: 30,
+    fontWeight: 500,
+  }
+
+  // Renders Select Option for each language available on Juicebox
+  const renderLanguageOption = (lang: string) => (
+    <Select.Option class="language-select-option" value={lang}>
+      <div>{Languages[lang].long}</div>
+    </Select.Option>
+  )
+
+  let currentSelectedLanguage = localStorage.getItem('lang') || 'en'
+
+  // Sets the new language with localStorage and reloads the page
+  function setLanguage(newLanguage: string) {
```

Comment:
**suggestion**: might be better to declare this as `const setLanguage = ()...` to avoid [hoisting](https://www.w3schools.com/js/js_hoisting.asp) weirdness.

Reply:
Good idea